src/css/components/card.scss

```diff
@@ -7,7 +7,20 @@
   border: 1px solid $color-textblack;
   border-radius: $rounded;
   margin-bottom: $grid-2;
-  padding: 0;
+  padding: 1.4rem $grid-2;
+
+  > .panel-row-is_clickable {
+    display: flex;
+  }
+
+  &:last-child {
+    margin-bottom: 0;
+    padding-bottom: $grid-2;
+  }
+
+  .panel-row:last-child {
```

Comment:
I like how you used `>` in the first rule above, can that be done here as well to keep more specific?